scripts/jenkins/Jenkinsfile

```diff
@@ -162,10 +162,8 @@ pipeline {
     post {
         always {
             script {
-                if (!env.BRANCH_NAME.startsWith("PR-")) {
                     // Send build notification
```

Comment:
Can you move the lines within the removed 'if' back a tab to keep proper spacing?